### PR TITLE
relap5 crash edits

### LINKED ISSUE #317: relap5 multideck issue
--------
Issue Description
--------
##### What did you expect to see happen?
the code should stop if one of the deck has failed

##### What did you see instead?
raven checks only for decks that successfully terminated

##### Do you have a suggested fix for the development team?
modification of the relap5 interface for a better check

##### Please attach the input file(s) that generate this error.  The simpler the input, the faster we can find the issue.

----------------
For Change Control Board: Issue Review
----------------
This review should occur before any development is performed as a response to this issue.
- [x] 1. Is it tagged with a type: defect or improvement? defect
- [x] 2. Is it tagged with a priority: critical, normal or minor? normal
- [x] 3. If it will impact requirements or requirements tests, is it tagged with requirements? no
- [x] 4. If it is a defect, can it cause wrong results for users? If so an email needs to be sent to the users. yes
- [x] 5. Is a rationale provided? (Such as explaining why the improvement is needed or why current code is wrong.) yes

-------
For Change Control Board: Issue Closure
-------
This review should occur when the issue is imminently going to be closed.
- [ ] 1. If the issue is a defect, is the defect fixed?
- [ ] 2. If the issue is a defect, is the defect tested for in the regression test system?  (If not explain why not.)
- [ ] 3. If the issue can impact users, has an email to the users group been written (the email should specify if the defect impacts stable or master)?
- [ ] 4. If the issue is a defect, does it impact the latest stable branch? If yes, is there any issue tagged with stable (create if needed)?
- [ ] 5. If the issue is being closed without a merge request, has an explanation of why it is being closed been provided?

### PR DESCRIPTION
**Pull request by mandd**
--------
Pull Request Description
--------
##### What issue does this change request address? (Use "#" before the issue to link it, i.e., #42.)
Closes #317 

##### What are the significant changes in functionality due to this change request?
realp5 interface

----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [x] 1. Review all computer code. ***ok***
- [x] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts). ***n/a***
- [x] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/raven/wiki/RAVEN-Code-Standards#python) for details. ***ok***
- [x] 4. Automated Tests should pass, including run_tests, pylint, manual building and xsd tests. If there are changes to Simulation.py or JobHandler.py the qsub tests must pass. ***they pass***
- [x] 5. If significant functionality is added, there must  be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large test. If new development on the internal JobHandler parallel system is performed, a cluster test must be added setting, in <RunInfo> XML block, the node ```<internalParallel>``` to True. ***not a significant functionality***
- [x] 6. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync. ***n/a***
- [x] 7. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done. ***#317***
- [x] 8. If an analytic test is changed/added is the the analytic documentation updated/added? ***n/a***

### CODE REVIEW

File: framework/CodeInterfaces/RELAP5/Relap5Interface.py [hunk 1]
```diff
@@ -107,16 +107,19 @@ def checkForOutputFailure(self,output,workingDir):
       @ Out, failure, bool, True if the job is failed, False otherwise
     """
     failure = True
-    errorWord = ["Transient terminated by end of time step cards","Transient terminated by trip"]
+    errorWord = ["Transient terminated by failure","Thermodynamic property error with minimum time step, transient being terminated"]
```
Comment (alfoa) [inline framework/CodeInterfaces/RELAP5/Relap5Interface.py]: If we just use the "goodWord" list now, we probably can delete the errorWord list.

Comment (mandd) [reply framework/CodeInterfaces/RELAP5/Relap5Interface.py]: I left it there as a note to discuss if it is worth to check error word as well as a redundancy measures. It surprises me this issue did not come out in the past.

**Comment by alfoa** [comment]
Comments addressed...

PR can be accepted.

Well done developer!

**Comment by mandd** [comment]
I made a small modification that should speed up the checking process. Let me know what do you think.

**Comment by alfoa** [comment]
did you open a PR?

**Comment by mandd** [comment]
oh no, i have added that comment but it was already merged
what I changed was in the PR already

**Comment by alfoa** [comment]
ok

**Comment by joshua-cogliati-inl** [comment]
Just for the record, did the last small modification get merged already, or does it need to be a new pull request?  Thanks.

**Comment by mandd** [comment]
the last modification was already included in the PR.
I added the comment prior refreshing my browser

**Pull request merged**